site/en/spec/glossary.md

```diff
@@ -58,11 +58,21 @@ See [Glyph](#glyph)
 
 ### TextLine
 
-A TextLine is a [region](#region) of text without line break.
+A text line is a single row of [words](#word) within a text [region](#region). (Depending on the region's or page's orientation, and the script's writing direction, it can be horizontal or vertical.)
+
+### Print space
+
+From the [PAGE-XML content schema documentation](https://ocr-d.de/de/gt-guidelines/pagexml/pagecontent_xsd_Complex_Type_pc_PrintSpaceType.html)
+
+> Determines the effective area on the paper of a printed page. Its size is equal for all pages of a book (exceptions: titlepage, multipage pictures).
+>
+> It contains all living elements (except marginals) like body type, footnotes, headings, running titles.
```

Comment:
```suggestion
> It contains all living elements (except marginalia) like body type, footnotes, headings, running titles.
```

Reply:
what is meant with "body type"?

Reply:
> what is meant with "body type"?

I don't know TBH. @tboenig this is your text, right? Does this come from [TEI/DTABf's body](http://deutsches-textarchiv.de/doku/basisformat/bodyAllg.html) by any chance?

Reply:
I think @tboenig just wanted to avoid a recursive definition ("printspace contains printspace+marginalia"). Let's just call that "paragraphs and headings" for simplicity.